server/server.go

```diff
@@ -147,9 +151,14 @@ func (svr *Server) HandleGet(w http.ResponseWriter, r *http.Request) error {
 		svr.WriteBadRequest(w, invalidDomain)
 		return err
 	}
+	commitmentType, err := ReadCommitmentType(r)
```

Comment:
Why are we reading a commit type here?

Reply:
The idea is that in the "naked" mode the plasma server does not interpret commitments passed to `GetBlob()` in any way, so they are just passed right through to the store. This is not the behavior in "optimism" mode.

Thinking more about it, I'm of the opinion that we we shouldn't offer a true naked mode but should instead return EigenDA-version encoded commitments, i.e. `append([]byte{byte(EigenDACommitmentEncodingVersion)}, rlpEncodedDACert...)`.